### DISCUSSION TITLE
Creating New List Fails when Using Brave Browser

### DISCUSSION THREAD

**Discussion by tgrecojs** [community/community#76972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Seems related to other issues some people are having (see https://github.com/orgs/community/discussions/64121#discussion-5531690)

I've included screenshot of the logs produced a moment ago when trying to add a list in Brave Browser. I decided to look into the `Greaselion` as I had never seen this before. Using Chat GPT-4 provided me with the following info:

> ... in the context of the Brave browser, "Greaselion" is a feature that enables developers to add custom scripts to specific web pages, akin to the idea of userscripts in Greasemonkey/Tampermonkey. This can be used to modify web pages on the fly, provide additional functionality, fix bugs, etc.
The ﻿GreaselianError you're seeing in your console would likely be associated with an issue or bug with these custom scripts.

After seeing that, I quickly swapped out Brave for the Firefox Dev Browser and found that this bug no longer occurred. Additionally, the lists that I created in Brave did show up (in other words, they were in fact created), so the bug is not that lists aren't created. It's that once a new list is created, the browser window hangs.

# Brave Screenshots

### Hanging Window
<img width="931" alt="brave-browser-github-lists-issue" src="https://github.com/community/community/assets/6646552/435b3329-c605-4f71-b949-701821591d36">

### Developer Console

![greaselion-logs](https://github.com/community/community/assets/6646552/199d4abb-de38-438c-b11f-96c6e65de953)

**Comment by jakechood** [comment]
Similar using Brave Browser on Windows 11

Steps to potentially repeat bug:
.......................
- via dropdown next to Bootstrap page star button
- Attempted to create a list named "Frameworks"
- Click "create" button
- [pop-up?] froze
- Refresh Page
.......................
- via dropdown next to Bootstrap page star button
- Attempted to create a list named "Frameworks"
- Click "create" button
- [pop-up?] froze
- *Click "Share feedback..." link
- Page refreshes behind [pop-up?] but frozen [pop-up?] remains
.......................
- via dropdown next to Bootstrap page star button
- *Opened list creation [pop-up?]
- Click "create" button
- [pop-up?] froze
- Click "Share feedback..." link
- Page refreshes behind [pop-up?] but frozen [pop-up?] remains

![Screenshot 2023-12-10 144152](https://github.com/community/community/assets/140123683/31300182-5bae-46fb-b8c6-ba874446e1c2)

**Reply by Afreenrubel** [reply]
9283608cc6b7be6b65a8e44983db012355fde4132009bf99d976b2f0896856a3

**Comment by PholphatPhumkaew** [comment]
Request Id: 532e215f-79d2-4ec5-a62f-aea3a0932800
Correlation Id: 34d001a8-db01-4be1-a3fd-9bae00f4b5f5
Timestamp: 2023-12-13T14:43:00Z
Message: AADSTS900561: The endpoint only accepts POST requests. Received a GET request.

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by lovelyMircho** [comment]
Github sa place

**Comment by ibrohim-qosimov** [comment]
I + jamim's comment

**Comment by Mlwody90n** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Seems related to other issues some people are having (see https://github.com/orgs/community/discussions/64121#discussion-5531690)
> 
> I've included screenshot of the logs produced a moment ago when trying to add a list in Brave Browser. I decided to look into the `Greaselion` as I had never seen this before. Using Chat GPT-4 provided me with the following info:
> 
> > ... in the context of the Brave browser, "Greaselion" is a feature that enables developers to add custom scripts to specific web pages, akin to the idea of userscripts in Greasemonkey/Tampermonkey. This can be used to modify web pages on the fly, provide additional functionality, fix bugs, etc.
> The ﻿GreaselianError you're seeing in your console would likely be associated with an issue or bug with these custom scripts.
> 
> After seeing that, I quickly swapped out Brave for the Firefox Dev Browser and found that this bug no longer occurred. Additionally, the lists that I created in Brave did show up (in other words, they were in fact created), so the bug is not that lists aren't created. It's that once a new list is created, the browser window hangs.
> 
> # Brave Screenshots
> 
> ### Hanging Window
> <img width="931" alt="brave-browser-github-lists-issue" src="https://github.com/community/community/assets/6646552/435b3329-c605-4f71-b949-701821591d36">
> 
> ### Developer Console
> 
> ![greaselion-logs](https://github.com/community/community/assets/6646552/199d4abb-de38-438c-b11f-96c6e65de953)
> 
> 
>

**Comment by le-dawg** [comment]
I want to add that I have the same issue with Safari 17.1 on Sonoma, M1.

List creation fails only client-side. Reloading the page shows that the list has indeed been created and the repo correctly signed to it.

**Comment by Afreenrubel** [comment]
Super
